src/main/java/spaceinvaders/group_22/ui/SpaceInvadersUI.java

```diff
@@ -15,17 +16,34 @@
  *
  */
 public class SpaceInvadersUI {
+	/**
+	 * Singleotn instance.
+	 */
+	private static SpaceInvadersUI instance;
```

Comment:
I'm not sure if this instance is completely thread safe.
The logger has a thread safe implementation, is there a reason why this implementation is not thread safe?